app/main.py

```diff
@@ -14,6 +16,8 @@
 app.mount("/static", StaticFiles(directory=STATIC_PATH), name="static")
 app.mount("/media", StaticFiles(directory=MEDIA_PATH), name="media")
 
+load_quotes.load_daily_quotes(next(get_db()))
```

Comment:
Prefer to use dependency injection using FastAPI's `Depends`

Reply:
As I understand, it is not possible to use "depends" outside a route function. In my case, I load the data at app initialization, regardless of any request. Therefore, I need to get a db-session, which is not from "depends"..

Reply:
Ah, I see. OK Then :)